classes/Http/Action/Reviewer/Talk/RateAction.php

```diff
@@ -28,15 +25,21 @@
      */
     private $talkHandler;
 
-    public function __construct(Talk\TalkHandler $talkHandler)
+    /**
+     * @var RequestValidator
+     */
+    private $requestValidator;
+
+    public function __construct(Talk\TalkHandler $talkHandler, RequestValidator $requestValidator)
     {
-        $this->talkHandler = $talkHandler;
+        $this->talkHandler      = $talkHandler;
+        $this->requestValidator = $requestValidator;
     }
 
     public function __invoke(HttpFoundation\Request $request): HttpFoundation\Response
     {
         try {
-            $this->validate($request, [
+            $this->requestValidator->validate($request, [
                 'rating' => 'required|integer',
```

Comment:
I'm unsure if we really need to throw a sophisticated validator at this problem, I must admit.

Reply:
We can probably just use a simpler check for this, but its good for this PR.